### PR TITLE
Ces 006 add api documentation for blog

### PR DESCRIPTION
**Pull request by Golderamit**

### CODE REVIEW

File: docs/blog/blog-data-dictionary.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#### **IsFeatured**
+
+
+      type: tinyint(1)
+      field name: is_published 
+
+    - title: PENDING
+      const: 0
+    - title: ACTIVE
+      const: 1
```
Comment (devrob-go) [inline docs/blog/blog-data-dictionary.md:55]: Do something like this:

IS_PUBLISHED_INACTIVE
IS_PUBLISHED_ACTIVE

File: docs/blog/blog-data-dictionary.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+#### **IsPublished**
+
+
+      type: tinyint(1)
+      field name: is_featured 
+
+    - title: PENDING
+      const: 0
+    - title: ACTIVE
+      const: 1
```
Comment (devrob-go) [inline docs/blog/blog-data-dictionary.md:67]: Same here:

IS_FEATURED_INACTIVE
IS_FEATURED_ACTIVE

File: docs/blog/create-blog.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+               "Category": "News",
+               "Content": "content is a",
+               "IsFeatured": "ACTIVE",
+               "ImageUrl": "https://demo.com/storage/images/coin-icons/default.jpg",
+               "IsPublished": "ACTIVE",
+               "Slug": "BdNews",
+	             "CreatedAt": "2023-05-08 09:12:15",
+	             "UpdatedAt": "2023-05-08 09:12:15",
+	             "CreatedBy": "123e4567-e89b-12d3-a456-426614174025",
+	             "UpdatedBy": "123e4567-e89b-12d3-a456-426614174025"
```
Comment (devrob-go) [inline docs/blog/create-blog.md:54]: maintain align.

File: docs/blog/create-blog.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+### Create Blog Response example:
+
+```json
+{
+  "Data": {
+        "Blog": {
+               "ID": "123e4567-e89b-12d3-a456-426614174111",
+	             "UserID": "123e4567-e89b-12d3-a456-426614174000",
```
Comment (devrob-go) [inline docs/blog/create-blog.md:42]: maintain align.

**Review by devrob-go — CHANGES_REQUESTED** [review]

File: docs/blog/blog-data-dictionary.md (hunk 2 above)
Comment (Golderamit) [reply docs/blog/blog-data-dictionary.md]: Updated

File: docs/blog/create-blog.md (hunk 3 above)
Comment (Golderamit) [reply docs/blog/create-blog.md]: Updated

File: docs/blog/create-blog.md (hunk 4 above)
Comment (Golderamit) [reply docs/blog/create-blog.md]: Updated

**Pull request merged**